src/test/java/com/liadov/cat/lesson10/task2/SausageControllerTest.java

```diff
@@ -0,0 +1,75 @@
+package com.liadov.cat.lesson10.task2;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * SausageControllerTest test for {@link SausageController}
+ *
+ * @author Aleksandr Liadov on 4/26/2021
+ */
+public class SausageControllerTest {
+
+    @Test
+    public void writeListToFileCorrectNumberOfObjects() throws IOException {
+        Path path = Paths.get("testFile.txt");
+        Files.deleteIfExists(path);
+        SausageController sausageController = new SausageController();
```

Comment:
У тебя в каждом тесте инициализируется SausageController. Давай это делать один раз Before class

Reply:
исправил на BeforeAll вместо Before class,
так как: 
@BeforeEach and @BeforeAll are the **JUnit 5** equivalents of @Before and @BeforeClass. These annotations were renamed with clearer names to avoid confusion.
согласно статье: https://www.baeldung.com/junit-before-beforeclass-beforeeach-beforeall

поправил остальные тестовые классы.
добавил удаление тестового файла с аннотацией @AfterAll